tools/vsimporter/xib2nib/UIStoryboardSegue.cpp

```diff
@@ -31,10 +31,13 @@ void UIStoryboardSegue::InitFromStory(XIBObject* obj) {
     ObjectConverter::InitFromStory(obj);
     _destination = getAttrAndHandle("destination");
     if (_destination) {
-        char szDestination[255];
-        sprintf(szDestination, "UIViewController-%s", _destination);
-        _destination = strdup(szDestination);
+        // IMPORTANT: Commenting out the next few lines because it this blocks segue handling when NIB filenames are prefixed with "UIViewController-".
+        //            This prefix is prepended generated under the IBTool (compatibility purposes) but it's not required by WinObjC-generated usage.
+        // char szDestination[255];
+        // sprintf(szDestination, "UIViewController-%s", _destination);
+        //_destination = strdup(szDestination);
```

Comment:
The right thing to do here is pull out the code. However, I'm was a little hesitant that there is a corner case or some usage I haven't accounted for. I'll pull it out - we can always restore it using source history.